tests/components/whirlpool/conftest.py

```diff
@@ -166,3 +169,23 @@ def mock_dryer_api():
     mock_dryer.get_cycle_status_spinning.return_value = False
     mock_dryer.get_cycle_status_washing.return_value = False
     return mock_dryer
+
+
+@pytest.fixture
+def mock_oven_api():
```

Comment:
```suggestion
def mock_oven_dual_cavity_api():
```

similar to the washer+dryer mocks, lets also add a `mock_oven_single_cavity_api` with the Upper only so that one also gets tested and snapshot.